src/postgres_deparse.c

```diff
@@ -130,173 +94,583 @@ isOp(const char *val)
 	return true;
 }
 
```

Comment:
Add an explanation here of how we use subparts and parts, and specifically how we flatten parts+subparts into just subparts when moving up in deparseStatePopStatement.